### DISCUSSION TITLE
Doc comment on Import data (/developers/academy/py/starter_text_data/text_collections/import_data)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1888] [Docs/blog comments]
# Doc comment on Import data (/developers/academy/py/starter_text_data/text_collections/import_data)

Weaviate stores data in "collections". A collection is a set of objects that share the same data structure. In our movie database, we might have a collection of movies, a collection of actors, and a collection of reviews.

https://weaviate.io/developers/academy/py/starter_text_data/text_collections/import_data

**Comment by homanmirgolbabaee** [comment]
batcher is cool !
